.ci_support/travis_.yaml

```diff
@@ -0,0 +1,4 @@
+MACOSX_DEPLOYMENT_TARGET:
```

Comment:
I'm out of the loop so bare with me as I try to understand the changes here. I though that variables like this one would live in a common config file fetched at build time, in the same fashion as conda-build-setup. Will these be define by the user for each feedstock? Or we are relying on conda-smithy to drop  these based on a common `conda_build_config.yaml` somewhere?

Reply:
`conda-smithy` is using the `conda_build_config.yaml` from `conda-forge-pinning` package when generating these files. This is a subset of the conda-forge-pinning file that conda-build renders for us and is used for separating the CI jobs. Otherwise we would be building for eg. different python versions in each CI job. We want one python version for one CI job.

Added advantage is that when pinning is changed in `conda-forge-pinning`, we can rerender the feedstock and check whether these files have changed as these files contain the bare minimum needed, and therefore an unrelated pinning change in `conda-forge-pinning` wouldn't affect these files.

Reply:
This is a recording of variables that were present at rendering time.  They're just a snapshot, and they "stick" until the next re-rendering.  That's when the common configuration comes in - at re-rendering time.

Reply:
Thanks for the explanation. We should try to write down some docs on this. These changes will trip many users.